client/extensions/woocommerce/state/sites/payment-methods/test/actions.js

```diff
@@ -7,37 +7,68 @@ import { spy } from 'sinon';
 /**
  * Internal dependencies
  */
-import { fetchPaymentMethods } from '../actions';
+import { fetchPaymentMethods, savePaymentMethod } from '../actions';
 import { LOADING } from 'woocommerce/state/constants';
 import useNock from 'test/helpers/use-nock';
 import { useSandbox } from 'test/helpers/use-sinon';
 import {
 	WOOCOMMERCE_PAYMENT_METHODS_REQUEST,
 	WOOCOMMERCE_PAYMENT_METHODS_REQUEST_SUCCESS,
+	WOOCOMMERCE_PAYMENT_METHOD_UPDATE,
+	WOOCOMMERCE_PAYMENT_METHOD_UPDATE_SUCCESS,
 } from 'woocommerce/state/action-types';
 
 describe( 'actions', () => {
+	useSandbox();
+	useNock( ( nock ) => {
+		nock( 'https://public-api.wordpress.com:443' )
+			.persist()
+			.get( '/rest/v1.1/jetpack-blogs/123/rest-api/' )
+			.query( { path: '/wc/v3/payment_gateways&_method=get' } )
+			.reply( 200, {
+				data: [ {
+					id: 'bacs',
+					title: 'Direct bank transfer',
+					description: 'Make your payment directly into our bank account.',
+					enabled: false,
+					method_title: 'BACS',
+					method_description: 'Allows payments by BACS, more commonly known as direct bank/wire transfer.',
+				} ]
+			} )
+			.post( '/rest/v1.1/jetpack-blogs/234/rest-api/' )
+			.query( { path: '/wc/v3/payment_gateways/paypal&_method=put' } )
+			.reply( 200, {
+				data: {
+					description: 'Pay via PayPal;',
+					enabled: false,
+					id: 'paypal',
+					method_description: 'PayPal Standard',
+					method_title: 'PayPal',
+					order: '',
+					settings: {
+						title: {
+							id: 'title',
+							label: 'Title',
+							value: 'PayPal'
+						},
+						email: {
+							id: 'email',
+							label: 'PayPal email'
+						},
+					},
+					title: 'PayPal7',
+				}
+			} )
+			.post( '/rest/v1.1/jetpack-blogs/456/rest-api/' )
+			.query( { path: '/wc/v3/payment_gateways/paypal&_method=put' } )
```

Comment:
Would you mind adding json: true to this so it doesn't fail after my change in #14650? Thanks.